### ISSUE TITLE
installation project missing on MSVC?

### ISSUE THREAD

**Issue by AlexanderBarnt** [unicorn-engine/unicorn#1992]
I was able to compile Unicorn 2.0.1.post1 using CMAKE with MSVC on Windows but I could not find any way to install it, looks like there is no "INSTALL" project in the solution:
<a href="https://imgbb.com/"><img src="https://i.ibb.co/kcSZg5w/devenv-nnlfg-Dj6u9.png" alt="devenv-nnlfg-Dj6u9" border="0" /></a>

I did enable the installation option in CMAKE:
<a href="https://ibb.co/ZNjRFb8"><img src="https://i.ibb.co/wYkZP2W/cmake-gui-p-Mxp7-YI1t3.png" alt="cmake-gui-p-Mxp7-YI1t3" border="0" /></a>

or maybe am i missing something?

**Comment by wtdcode** [comment]
Unfortunately, I don't see reasonable ways to "install" a package on Windows natively so `UNICORN_INSTALL` only applies on *nix systems. However, `vcpkg` is known to work this out and you may have a try.

**Comment by AlexanderBarnt** [comment]
its suppose that the latest Visual studio comes with vcpkg integrated, sorry for asking this but I am not used to it, can I use it? I just want the compiled files, headers, etc from my build. I went to the download page and I tried to download the "Windows core engine" but both 32 and 64 links say "not found".

**Comment by wtdcode** [comment]
> its suppose that the latest Visual studio comes with vcpkg integrated, sorry for asking this but I am not used to it, can I use it? I just want the compiled files, headers, etc from my build. I went to the download page and I tried to download the "Windows core engine" but both 32 and 64 links say "not found".
> 
> edit: any posibility to just add a INSTALL project in the solution like in most of the projects? that would save a lot of time from people.

Does `INSTALL` really works on Windows? I think I need to read more docs about cmake.

**Comment by AlexanderBarnt** [comment]
> > its suppose that the latest Visual studio comes with vcpkg integrated, sorry for asking this but I am not used to it, can I use it? I just want the compiled files, headers, etc from my build. I went to the download page and I tried to download the "Windows core engine" but both 32 and 64 links say "not found".
> > edit: any posibility to just add a INSTALL project in the solution like in most of the projects? that would save a lot of time from people.
> 
> Does `INSTALL` really works on Windows? I think I need to read more docs about cmake.

yeah, it just copies the right files (lib, dll, headers etc) depending on the build into the install directory

**Comment by vlasta-labsky** [comment]
I discovered this issue too.
In my build workflow on github I needed to add
`(Get-Content CMakeLists.txt).Replace('if(UNICORN_INSTALL AND NOT MSVC)', 'if(UNICORN_INSTALL)') | Set-Content CMakeLists.txt` to make unicorn behave as any other cmake project on windows.
I dont use vcpkg, but manage all free software on windows by means of pkgconfig.
I did it like this long before any vcpkg even existed.
It would be very nice to enable installation on windows so I could remove this hardcoded string replacement from my scripts.
